README.rst

```diff
@@ -1,6 +1,25 @@
 sentinelsat
 ===========
 
+⚠️ Sentinelsat is currently not functional for downloading from official mirrors ⚠️
```

Comment:
```suggestion
⚠️ Sentinelsat is currently not functional for downloading from the CDSE ⚠️
```

Reply:
I would lump the national mirrors under "official mirrors", so that wording seems a bit misleading to me.